### PR TITLE
pyqt-distutils-0.7.3 version bump, py3_{7..9} port

### PR DESCRIPTION
**Pull request by LebedevV**
Related to the Bugzilla bug 719428 
Successfully compiles&installs for all python versions mentioned.
Repoman -d full passed for portage-3.0.4-r1, repoman-3.0.1

Bug: https://bugs.gentoo.org/719428
Closes: https://bugs.gentoo.org/719428

### CODE REVIEW

**Comment by LebedevV** [comment]
Signoff added

File: dev-python/pyqt-distutils/pyqt-distutils-0.7.3.ebuild [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6..9} )
 inherit distutils-r1
 
 DESCRIPTION="distutils extension to work with PyQt applications and UI files"
```
Comment (thesamesam) [inline dev-python/pyqt-distutils/pyqt-distutils-0.7.3.ebuild:9]: Can't get github to get right line, but the explicit setuptools dep shouldn't be needed anymore.

Comment (LebedevV) [reply dev-python/pyqt-distutils/pyqt-distutils-0.7.3.ebuild:10]: Seems to be done, thank you for pointing it

**Comment by LebedevV** [comment]
pytest added, tests pass for all mentioned python version.
SRC_URI changed to the corresponding GitHub archive, as far as there is no "tests" folder in the PyPI version

File: dev-python/pyqt-distutils/pyqt-distutils-0.7.3.ebuild [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
-DEPEND="
-	dev-python/setuptools[${PYTHON_USEDEP}]
-"
+DISTUTILS_USE_SETUPTOOLS=rdepend
```
Comment (thesamesam) [inline dev-python/pyqt-distutils/pyqt-distutils-0.7.3.ebuild:21]: We pop this above the inherit, like [here](https://dev.gentoo.org/~mgorny/python-guide/distutils.html#different-build-system-variations).

**Comment by thesamesam** [comment]
Merged! Thank you for your contribution.

I have squashed the commits, adjusted the commit messages a little, and made some tiny tidy changes too, if you want to see. Thanks again!

**Comment by LebedevV** [comment]
Many thanks!

**Pull request closed without merging**